### PR TITLE
Setup neptune-detectron2 e2e tests 

### PR DESCRIPTION
**Pull request by AleksanderWWW**

### CODE REVIEW

File: .github/workflows/integrations.yml [hunk 1]
```diff
@@ -12,316 +17,362 @@ env:
...
-          python-version: ${{ matrix.python-version }}
-          os: ${{ matrix.os }}
-          module: zenml
-          report_job: 'zenml (${{ matrix.os }} - py${{ matrix.python-version }})'
+          working_directory: neptune-detectron2
 
-  zenml-notify:
-    needs: [zenml]
+  detectron2-notify:
+    needs: [ neptune ]
```
Comment (Raalsky) [inline .github/workflows/integrations.yml:375]: `detectron2`

File: .github/workflows/integrations.yml [hunk 2]
```diff
@@ -1,6 +1,11 @@
 name: integrations
 
 on:
+  push:
```
Comment (Raalsky) [inline .github/workflows/integrations.yml:4]: This was only for testing purposes, let's remove it :wink:

**Comment by Raalsky** [comment]
CHANGELOG 😉

**Comment by AleksanderWWW** [comment]
@Raalsky How about now?

**Comment by Raalsky** [comment]
Meh, my mistake. It shouldn't be a part of a CHANGELOG. It was definitely too early in the morning for me :P. Could you revert the last commit? :D. I'm really really sorry about that!

**Review by Raalsky — APPROVED** [review]

**Pull request merged**